src/lumigo_tracer/lambda_tracer/lambda_reporter.py

```diff
@@ -46,6 +48,9 @@
     int(os.environ.get("LUMIGO_MAX_SIZE_FOR_REQUEST_ON_ERROR", 1024 * 990)), REQUEST_MAX_SIZE
 )
 MAX_NUMBER_OF_SPANS: int = int(os.environ.get("LUMIGO_MAX_NUMBER_OF_SPANS", 2000))
+
+# Size of spans sent that is kept for the enrichment span additional info added during sending
+SPANS_SEND_SIZE_ENRICHMENT_SPAN_BUFFER = 200
```

Comment:
Why 200?

Reply:
200 is a const large enough to add
```
dropped_spans_reasons[DroppedSpansReasons.SPANS_SENT_SIZE_LIMIT.value] = {
            "drops": len(msgs) - len(spans_dict),
        }
```

I don't think we need to over complicate it with a dynamic buffer or something like that